modules/account-lib/src/coin/dot/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { BaseUtils } from '../baseCoin';
+import { NotImplementedError } from '../baseCoin/errors';
```

Comment:
If we are going to encode and decode address with keyring, please add unimplemented methods for encode and decode.
https://polkadot.js.org/docs/keyring/start/ss58#how-is-the-encoding-done